services/simcore/docker-compose.deploy.local.yml

```diff
@@ -21,7 +21,7 @@ services:
       - SSL_CERT_FILE=/usr/local/share/ca-certificates/osparc.crt
   director:
     deploy:
-      replicas: 1
+      replicas: ${SIMCORE_DIRECTOR_V0_REPLICAS}
     environment:
       # needed to pass the self-signed certificate to the spawned services
```

Comment:
Forgot `SIMCORE_DASK_SCHEDULER_REPLICAS` for local compose?

Reply:
lgtm as discussed personally :--) much appreciated